components/03-modules/gallery/Gallery.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div :class="['gallery', horizontal ? 'gallery--horizontal' : 'gallery--vertical']">
+    <div :class="['gallery__stage', horizontal ? 'gallery__stage--horizontal' : 'gallery__stage--vertical']">
+      <div class="gallery__frame">
+        <alpaca-badge
+          v-if="productLabel"
+          :class="productLabel.class"
+        >
+          {{ productLabel.text }}
+        </alpaca-badge>
+        <alpaca-image
+          :src="imageMain.dataSrc"
+          :alt="imageMain.alt"
+        />
+      </div>
+    </div>
+    <div :class="['gallery__nav', horizontal ? 'gallery__nav--horizontal' : 'gallery__nav--vertical']">
+      <div class="gallery__thumb-arr">
+        <alpaca-icon
+          icon="angle-up"
+          :class="['gallery__icon-arrow', horizontal ? 'gallery__icon-arrow--horizontal' : 'gallery__icon-arrow--vertical']"
+        />
+      </div>
+      <div
+        v-for="(thumb, i) in thumbs"
+        :key="i"
+        :class="['gallery__thumb', thumb.imageThumb.active && 'gallery__thumb--active']"
+      >
+        <alpaca-image
+          :src="thumb.imageThumb.dataSrc"
+          :alt="thumb.imageThumb.alt"
+        />
+      </div>
+      <div class="gallery__thumb-arr">
+        <alpaca-icon
+          icon="angle-down"
+          :class="['gallery__icon-arrow', horizontal ? 'gallery__icon-arrow--horizontal' : 'gallery__icon-arrow--vertical']"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  import AlpacaBadge from '../../02-elements/badge/Badge.vue'
+  import AlpacaImage from '../../02-elements/image/Image.vue'
+  import AlpacaIcon from '../../01-globals/icon/Icon.vue'
+
+  export default {
+    components: {
+      AlpacaBadge,
+      AlpacaImage,
+      AlpacaIcon
+    },
+    props: {
+      imageMain: {
+        type: Object,
+        required: true
+      },
+      thumbs: {
```

Comment:
IMO we should get just one array of images, but in two sizes. Clicking on the thumbnail should change the main image.